src/Http/Controllers/SwaggerController.php

```diff
@@ -70,9 +71,10 @@ public function api()
      * Display Oauth2 callback pages.
      *
      * @return string
+     * @throws L5SwaggerException
```

Comment:
This will not throw `L5SwaggerException`

Reply:
Correct me if I'm wrong but it's throw with the helper `swagger_ui_dist_path` here `src/helpers.php`

Reply:
Yup. I forgot about that.